Neos.Flow/composer.json

```diff
@@ -32,6 +31,7 @@
         "doctrine/orm": "~2.5.0",
         "doctrine/migrations": "~1.3.0",
         "doctrine/dbal": "~2.5.0",
+        "doctrine/common": ">=2.4,<2.8-dev",
```

Comment:
Should be `2.8.x-dev` AFAIK - it works, but there is a difference, see https://semver.mwl.be/#?package=doctrine%2Fcommon&version=%3E%3D2.4,%3C2.7-dev&minimum-stability=stable vs >=2.4,<2.7.x-dev (using 2.7 there to make the difference visible). What is the intended result?